packages/stark-ui/base.spec.ts

```diff
@@ -21,13 +19,13 @@ if (window.NodeList && !NodeList.prototype.forEach) {
 }
 
 /* tslint:disable:no-import-side-effect */
-import "zone.js/dist/zone";
-import "zone.js/dist/long-stack-trace-zone";
-import "zone.js/dist/proxy"; // since zone.js 0.6.15
-import "zone.js/dist/sync-test";
-import "zone.js/dist/jasmine-patch"; // put here since zone.js 0.6.14
-import "zone.js/dist/async-test";
-import "zone.js/dist/fake-async-test";
+import "../../node_modules/zone.js/dist/zone";
+import "../../node_modules/zone.js/dist/long-stack-trace-zone";
+import "../../node_modules/zone.js/dist/proxy"; // since zone.js 0.6.15
+import "../../node_modules/zone.js/dist/sync-test";
+import "../../node_modules/zone.js/dist/jasmine-patch"; // put here since zone.js 0.6.14
+import "../../node_modules/zone.js/dist/async-test";
+import "../../node_modules/zone.js/dist/fake-async-test";
```

Comment:
Could you add a FIXME comment here? you can reference https://github.com/angular/angular/issues/29531

Reply:
Well, in fact, this is not related to Angular, it is just the fact that we now have `core-js` as a direct dependency in our root package, before we didn't and it was just working fine by coincidence because `core-js` was a transient dependency.

So it is better like this now regardless of which version of `core-js` is used.

Reply:
Mmm I just saw you were talking about `zone.js`. Well, it is the same reason, we explicitly point to the package at the root which we already have as a direct dependency 😉 I think it is safer this way and easier to troubleshoot

Reply:
I think it would be better that we add zone.js and core.js to our `tsconfig.spec.json`since we already do this for some other dependencies.
-----------------------------------
Then something not really related to this PR....
If we add something like 
```
"paths": {
    "*": ["../stark-core/node_modules/*", "../../node_modules/*", ...],
}
```
We should hit all these indirect dependencies in one go. But we should investigate if it has a significant impact on our build process.

Also I think we should review our dependency management / use of transient dependencies. But I will move that discussion to another issue. 😄